packages/cli/src/commands/logout.ts

```diff
@@ -0,0 +1,19 @@
+import { APICommand } from '@smartthings/cli-lib'
+
+
+export default class LogoutCommand extends APICommand {
+	static flags = APICommand.flags
+
+	async run(): Promise<void> {
+		const { args, argv, flags } = this.parse(LogoutCommand)
+		await super.setup(args, argv, flags)
+
+		if (this.token) {
+			this.log(`Profile ${this.profileName} is set up using a PAT.`)
+			this.exit()
+		}
+
+		this.authenticator.logout && this.authenticator.logout()
```

Comment:
We should either `await` here or make logout sync.

Reply:
Fixed